R/dist_spec.R

```diff
@@ -549,75 +565,105 @@ print.dist_spec <- function(x, ...) {
 #' Note that currently uncertainty in distributions is not plot.
```

Comment:
I guess this no longer applies.

Reply:
Fixed in 2aadbbd6525b146089d3aeb9602e665713d1b245